contracts/RenPool.sol

```diff
@@ -61,6 +64,14 @@ contract RenPool {
         // TODO: register pool into RenPoolStore
     }
 
+    modifier onlyAdmin() {
```

Comment:
I think we should rename admin to Node Operator, a bit like RocketPool does. This we we don't have the confusion between owner and admin

Reply:
yes, good point!